docs/navigation-parameters-reference.md

```diff
@@ -338,6 +350,165 @@ settings: {
+Web components can communicate over an event bus.
+- **attributes**:
+  - **renderer**:
+    - **type**: object
+    - **description**: meta information about the layout of the compound web components
+    - **attributes**:
+      - **use**:
+        - **type**: string OR object
+        - **description**: You can define a css layout, like the css `grid` or implement an extended layout by defining an object using the `extends` property. In that case you have the possibility to implement the following functions the manipulate the standard renderer.
+        - **attributes**:
+          - **extends**
+            - **type**: string, e.g. `grid`
+            - **description**: 
+          - **createCompoundContainer**
+            - **type**: function
+            - **description**: This function gets the grid layout `config` object as parameter. 
+          - **createCompoundItemContainer**
+            - **type**: function
+            - **description** This function gets a the grid layout `config` object and the layout config object for the item.
+          - **attachCompoundItem**
+            - **type**: function
+            - **description** This function allows you to attach custom html to the item. This function gets the whole html container of the compound items and the item container as parameter.
+      - **config**:
+        - **type**: object
+        - **description**: defines the configuration object of the grid layout
+        - **attributes**:
+          - **columns**: represents the css `grid-template-columns`, e.g `1fr 2fr` .
+          - **rows**: represents the css `grid-template-rows`, e.g. `150px 150px`.
+          - **gap**: represents the css `grid-gap`, e.g. `auto`.
+          - **min-height** min height
+          - **layouts**
+            - **type**: array
+            - **description**: defines the configuration objects of the grid layout for media queries.
+            - **attributes**:
+              - **columns**: represents the css `grid-template-columns`, e.g `1fr 2fr` .
+              - **rows**: represents the css `grid-template-rows`, e.g. `150px 150px`.
+              - **gap**: represents the css `grid-gap`, e.g. `auto`.
+              - **min-width** min width
+              - **max-width** max width
+  - **children**
+    - **type**: array
+    - **description**: Array of web component nodes.
+    - **attributes**:
+      - **id**: unique `id` of the web component.
+      - **viewUrl**: url which points to the web component js file.
+      - **context**: object, which you can pass to the web component.
+      - **layoutConfig**: config object to define the position of an item in a grid. The properties are `row` and `column` and gets the same values as in the css grid standard. If you want to use the mechanism of nested web components you can define a `slot` property with the slot name instead of the config object. In that case this web component node will plugged in the parent web component.
```

Comment:
```suggestion
      - **id**: unique `id` of the web component.
      - **viewUrl**: URL which points to the web component `.js` file.
      - **context**: object, which you can pass to the web component.
      - **layoutConfig**: config object to define the position of an item in a grid. The properties are `row` and `column` and get the same values as in the CSS grid standard. If you want to use the mechanism of nested web components, you can define a `slot` property with the slot name instead of the config object. In that case this web component node will be plugged in the parent web component.
```